README.md

```diff
@@ -106,7 +106,7 @@ Read more about how to install the example webhook [here](deploy/kubernetes/webh
 
 * `--leader-election-namespace <namespace>`: The namespace where the leader election resource exists. Defaults to the pod namespace if not set.
 
-* `--metrics-address`: The TCP network address where the prometheus metrics endpoint will run (example: `:8080` which corresponds to port 8080 on local host). The default is empty string, which means metrics endpoint is disabled.
+* `--metrics-address`: The TCP network address where the prometheus metrics endpoint and leader election health check will run (example: `:8080` which corresponds to port 8080 on local host). The default is empty string, which means metrics endpoint and leader election check are disabled.
```

Comment:
Can you also incorporate the changes that @verult did in https://github.com/kubernetes-csi/external-attacher/pull/279?

Reply:
This contains the latest changes around the HTTP endpoint: https://github.com/kubernetes-csi/external-provisioner/pull/537

Reply:
Done